includes/class-nrh.php

```diff
@@ -17,10 +17,60 @@ class NRH {
 	 * Add hooks.
 	 */
 	public static function init() {
+		add_filter( 'newspack_donation_checkout_url', [ __CLASS__, 'redirect_to_nrh_checkout' ], 10, 3 );
 		add_filter( 'newspack_blocks_donate_block_html', [ __CLASS__, 'handle_custom_campaign_id' ], 10, 2 );
-		add_filter( 'googlesitekit_gtag_opt', [ __CLASS__, 'googlesitekit_gtag_opt' ], 11 );
-		add_filter( 'googlesitekit_amp_gtag_opt', [ __CLASS__, 'googlesitekit_amp_gtag_opt' ], 11 );
-		add_filter( 'newspack_blocks_donate_block_html', [ __CLASS__, 'render_nrh_donate_block' ], 10, 2 );
+		add_filter( 'allowed_redirect_hosts', [ __CLASS__, 'allow_redirect_to_nrh' ] );
```

Comment:
The cross-domain linker is removed because applying the recent Donate block changes to this alternative implementation is too big a lift, and the NRH integration will soon be updated and not rely on a redirect anymore. This is a regression, but a planned one.

Reply:
Sounds good—thanks for explaining.